tcms/testcases/views.py

```diff
@@ -1696,7 +1697,7 @@ def tag(request):
                     return HttpResponse(json.dumps(ajax_response))
         return HttpResponse(json.dumps(ajax_response))
 
-    form = CaseTagForm(initial={'tag': request.REQUEST.get('o_tag')})
+    form = CaseTagForm(initial={'tag': request.POST.get('o_tag')})
```

Comment:
These changes are related to the issue replacing ``request.REQUEST`` that I'm working on. I'll fix it later. I think you can remove these changes from this PR.